Examples/ByFeature/FunctionalInjection/Stock.feature

```diff
@@ -1,6 +1,6 @@
 ﻿Feature: Refunded or replaced items should be returned to stock
 
-Scenario: Refunded items should be returned to stock
+Scenario 1: Refunded items should be returned to stock
```

Comment:
Not sure this makes sense in terms of the intention of the Gherkin design. Could just suggest using `Scenario: 1. ...` and not encourage people to get into rippling auto-numbering edits

Reply:
I would prefer to separate it from this pull request. @michalkovy is correct that all blog posts shows this particular usage. We can decide that we do not want to support it anymore, but it can be easily driven as separate pull request (the change is small)